nmostesting/GenericTest.py

```diff
@@ -383,6 +397,97 @@ def validate_schema(self, payload, schema):
     def do_request(self, method, url, **kwargs):
         return TestHelper.do_request(method=method, url=url, **kwargs)
 
+    def make_key_cert_files(self, cert_file, key_file):
+        # create a key pair
+        k = crypto.PKey()
+        k.generate_key(crypto.TYPE_RSA, 2048)
+
+        # create cert
+        cert = crypto.X509()
+        cert.set_version(2)
+        cert.get_subject().C = "GB"
+        cert.get_subject().ST = "England"
+        cert.get_subject().O = "NMOS Testing Ltd"  # noqa: E741
+        ca_cert_subject = cert.get_subject()
+        ca_cert_subject.CN = "ca.testsuite.nmos.tv"
+        cert.set_issuer(ca_cert_subject)
+        cert.get_subject().CN = "mocks.testsuite.nmos.tv"
+        cert.set_serial_number(x509.random_serial_number())
+        cert.gmtime_adj_notBefore(0)
+        cert.gmtime_adj_notAfter(10*365*24*60*60)
+        cert.set_pubkey(k)
+        # get Root CA key
+        capkey = open(CONFIG.KEY_TRUST_ROOT_CA, "r").read()
+        ca_pkey = crypto.load_privatekey(crypto.FILETYPE_PEM, capkey)
+        # get Root CA cert
+        cacert = open(CONFIG.CERT_TRUST_ROOT_CA, "r").read()
+        ca_cert = crypto.load_certificate(crypto.FILETYPE_PEM, cacert)
+        # create cert extension
+        san = ["DNS:mocks.{}".format(CONFIG.DNS_DOMAIN), "DNS: nmos-mocks.local"]
```

Comment:
One has `DNS:` without a following space and the other one does have a following space. Does that matter? I'm not sure it is possible to use/test auth with mDNS, but have you tried?

Reply:
I have tested with mDNS and unicast DNS, let's make them the same style

Reply:
Fixed